source/class/qx/tool/compiler/jsdoc/Parser.js

```diff
@@ -99,7 +99,7 @@ qx.Class.define("qx.tool.compiler.jsdoc.Parser", {
           try {
             cmd.body = converter.makeHtml(cmd.body);
           } catch (e) {
-            qx.tool.compiler.Console.error(`Markdown conversion error: "${e.message}" found in \n${cmd.body.trim()}`);
+            qx.tool.compiler.Console.warn(`Markdown conversion: "${e.message}" found in \n${cmd.body.trim()}`);
```

Comment:
Isn't the exception being raised a bug in the "showdown" module?  EG what is wrong with having a comment like 

```
    /**
     * --------------------------------------------------------------------------
     * [Constructor]
     * --------------------------------------------------------------------------
     */
    construct: function()
```

Even if we decided to enforce strict rules on comment formatting, it should not say something inexplicable just `url.replace is not a function`.

Reply:
Maybe the simplest thing would be making the warning more explicative: 
```
Markdown conversion problem: Error "${e.message}" was thrown parsing "${cmd.body.trim()}". Please review your doc comments for Markdown compatibility.
```

Reply:
I don't think there should be *anything* in a comment that causes it to barf. If it's understandable as markdown, then treat it as such. If not, don't tell the user how to write a comment. (A warning that can be turned off via command-line option or `@ignore` or whatever is ok; just not an error.)